R/app.R

```diff
@@ -1,167 +1,215 @@
+options(shiny.autoreload = TRUE)
+options(shiny.launch.browser = .rs.invokeShinyWindowExternal)
+
```

Comment:
Don't include these

Reply:
Good spot. That was for dev purposes. Had meant to take them out.